src/main/java/dev/jbang/catalog/Template.java

```diff
@@ -14,11 +14,14 @@
 	@SerializedName(value = "file-refs")
 	public final Map<String, String> fileRefs;
 	public final String description;
+	public final Map<String, TemplateProperty> properties;
 
-	public Template(Map<String, String> fileRefs, String description, Catalog catalog) {
+	public Template(Map<String, String> fileRefs, String description, Catalog catalog,
```

Comment:
This is not wrong of course, but could you change the order a bit? The `catalog` is meant to be the last argument in the list.